engagement_db_to_analysis.py

```diff
@@ -0,0 +1,37 @@
+import argparse
+import importlib
+
+from core_data_modules.logging import Logger
+
+from src.engagement_db_to_analysis.engagement_db_to_analysis import get_project_messages_from_engagement_db
+
+
+log = Logger(__name__)
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Runs the engagement to analysis phases of the pipeline")
+
+    parser.add_argument("user", help="Identifier of the user launching this program")
+    parser.add_argument("google_cloud_credentials_file_path", metavar="google-cloud-credentials-file-path",
+                        help="Path to a Google Cloud service account credentials file to use to access the "
+                             "credentials bucket"),
+    parser.add_argument("engagement_db_datasets_export_dir", metavar="engagement-db-datasets-export-dir",
```

Comment:
We made it optional before because we had some datasets that did not have the last updated field in the messages which is not the case now. I don't see the benefit of making incremental download optional. 

Updated the folder to generic cache.